song_rater/views.py

```diff
@@ -13,5 +15,30 @@ class SongCreate(CreateView):
     success_url = '/song-added'
 
 
+class SongList(ListView):
+    model = Song
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        song_table = []
+
+        for song in Song.objects.all().prefetch_related('ratings'):
```

Comment:
not really a comment about this section of code, but I think you'll like [this article](https://docs.djangoproject.com/en/3.1/topics/db/optimization/) in the django docs.

Reply:
Thank you for sharing. Indeed interesting.